### PR TITLE
fix: dev: Parametrize setup script path.

### PR DESCRIPTION
**Pull request by saenzpa**

### CODE REVIEW

File: lib/topology_docker_openswitch/openswitch.py [hunk 1]
```diff
@@ -182,9 +182,13 @@ def _setup_system(self):
         """
 
         # Write and execute setup script
-        with open(
-            join(dirname(normpath(abspath(__file__))), 'openswitch_setup')
-        ) as openswitch_setup_file:
+        if script_path is not None:
```
Comment (ocelotl) [inline lib/topology_docker_openswitch/openswitch.py]: May I recommend this:

```
>>> script_file = None or 3
>>> script_file
3
>>> script_file = 4 or 3
>>> script_file
4
```

File: lib/topology_docker_openswitch/openswitch.py [hunk 2]
```diff
@@ -162,17 +162,17 @@ def __init__(
             )
         )
 
-    def notify_post_build(self):
+    def notify_post_build(self, script_path=None):
```
Comment (ocelotl) [inline lib/topology_docker_openswitch/openswitch.py:165]: Please document ``script_path``.

File: lib/topology_docker_openswitch/openswitch.py [hunk 3]
```diff
@@ -162,17 +162,17 @@ def __init__(
...
         reached.
 
         See :meth:`DockerNode.notify_post_build` for more information.
         """
         super(DockerOpenSwitch, self).notify_post_build()
-        self._setup_system()
+        self._setup_system(script_path)
 
-    def _setup_system(self):
+    def _setup_system(self, script_path):
```
Comment (ocelotl) [inline lib/topology_docker_openswitch/openswitch.py]: Let's add a default value to ``script_path`` of ``None``.

File: lib/topology_docker_openswitch/openswitch.py [hunk 4]
```diff
@@ -182,9 +182,13 @@ def _setup_system(self):
         """
 
         # Write and execute setup script
-        with open(
-            join(dirname(normpath(abspath(__file__))), 'openswitch_setup')
-        ) as openswitch_setup_file:
+        if script_path is not None:
+            script_file = script_path
+        else:
+            script_file = join(
```
Comment (ocelotl) [inline lib/topology_docker_openswitch/openswitch.py]: Let's rename ``script_file`` to ``openswitch_setup_path``.

File: lib/topology_docker_openswitch/openswitch.py (hunk 2 above)
Comment (saenzpa) [reply lib/topology_docker_openswitch/openswitch.py:165]: Ready

**Pull request merged**